src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -515,13 +515,19 @@ std::vector<SMALL_RECT> Terminal::_GetSelectionRects() const
     }
 
     // Add anchor offset here to update properly on new buffer output
-    SHORT temp1, temp2;
-    THROW_IF_FAILED(ShortAdd(_selectionAnchor.Y, _selectionAnchor_YOffset, &temp1));
-    THROW_IF_FAILED(ShortAdd(_endSelectionPosition.Y, _endSelectionPosition_YOffset, &temp2));
+    SHORT y1;
+    SHORT y2;

```

Comment:
I'd probably prefer more descriptive names for `y1`, `y2`, `x1`, and `x2`

Reply:
Refactored so now we don't even need these temp variables :)